src/test/java/com/nvidia/spark/rapids/jni/TimeZoneTest.java

```diff
@@ -44,10 +44,10 @@ static void cleanup() {
   @Test
   void databaseLoadedTest() {
     // Check for a few timezones
-    List transitions = GpuTimeZoneDB.getHostFixedTransitions("UTC+8");
+    List transitions = GpuTimeZoneDB.getHostTransitions("UTC+8");
     assertNotNull(transitions);
     assertEquals(1, transitions.size());
-    transitions = GpuTimeZoneDB.getHostFixedTransitions("Asia/Shanghai");
+    transitions = GpuTimeZoneDB.getHostTransitions("Asia/Shanghai");
     assertNotNull(transitions);
     ZoneId shanghai = ZoneId.of("Asia/Shanghai").normalized();
     assertEquals(shanghai.getRules().getTransitions().size() + 1, transitions.size());
```

Comment:
Please add a check for a non-fixed timezone here. Should verify that the cached rule was created.